packages/plugin-core/src/test/suite-integ/components/pods/BaseExportPodCommand.test.ts

```diff
@@ -72,7 +79,7 @@ suite("BaseExportPodCommand", function () {
             exportScope: PodExportScope.Hierarchy,
           });
 
-          // 'foo' note and its child:
+          // 'foo' note and its child: foo.test should not appear
```

Comment:
nit: could you add this part to the test description, i.e.

"THEN hierarchy note props should be in the export payload AND a note with a hierarchy match but in a different vault should not appear"